hosts

```diff
@@ -95,10 +95,10 @@ builder7.rht.gluster.org
 builder8.rht.gluster.org
 builder9.rht.gluster.org
 
-
 [jenkins_builders:children]
 jenkins_builders_rht
 jenkins_builders_rackspace
+bugziller
```

Comment:
We have two options. Trim down what a base jenkins builder env means. Or not care. I'm going with not care for now :)

I need some of the things, but not all of them.